micro/DRAM.cpp

```diff
@@ -32,20 +32,16 @@ class MicroBenchDRAM {
   // Since we cannot use explicit memory operations to initialize the input buffer,
   // we have to keep this around, unfortunately.
   std::vector<DataT> input;
```

Comment:
If passing a `const DataT*` pointer forces a copy (as we discussed yesterday), you could also get rid of this here and only temporarily create it in `setup()`.